### PR TITLE
modified the create database statements according to connection_args given in the ..._handler.py files for each database wherever applicable

### PR DESCRIPTION
**Pull request by martyna-mindsdb**
Fixes [#37 from mindsdb_private](https://github.com/mindsdb/mindsdb_private/issues/37)

## Please describe what changes you made, in as much detail as possible
  - modified the `create database` statements according to `connection_args` given in the [..._handler.py files](https://github.com/mindsdb/mindsdb/tree/staging/mindsdb/integrations/handlers) for each database wherever applicable
[DATABASE - MindsDB Documentation.pdf](https://github.com/mindsdb/mindsdb/files/9500037/DATABASE.-.MindsDB.Documentation.pdf)

mindsdb

### CODE REVIEW

**Comment by Ricram2** [comment]
@StpMax I would love your review on this one, please :)

**Comment by StpMax** [comment]
1. let change `Mimzo3i-mxt@9CpThpBj` to just `password`.
2. Handlers may be changed and therefore connection string may also be changed in any time. May be add note like
```
Most handlers is supported by community. To get actual info about connection, please check readme.md in handler's folder.
```

**Review by StpMax — APPROVED** [review]

**Comment by martyna-mindsdb** [comment]
@StpMax 
I will modify all passwords to `password`.
About the `readme.md` file, I noticed that not all the handler folders have this file. The following databases are missing the `readme.md` file: mariadb, mssql, mysql, postgres, questdb, singlestore, snowflake, tidb, trino.

**Comment by martyna-mindsdb** [comment]
Here is the `SQL API -> CREATE -> DATABASE` doc page:
[DATABASE - MindsDB Documentation.pdf](https://github.com/mindsdb/mindsdb/files/9528883/DATABASE.-.MindsDB.Documentation.pdf)

**Review by Ricram2 — APPROVED** [review]

File: docs/mindsdb-docs/docs/sql/create/databases.md [hunk 1]
```diff
@@ -102,540 +102,1240 @@ The list of databases supported by MindsDB keeps growing. Here are the currently
...
+    ```
+
+=== "Example"
+
+    ```sql
+    CREATE DATABASE amazons3_datasource
+    WITH ENGINE='s3',
+    PARAMETERS={
+        "aws_access_key_id": "PCAQ2LJDOSWLNSQKOCPW",
+        "aws_secret_access_key": "U/VjewPlNopsDmmwItl34r2neyC6WhZpUiip57i",
```
Comment (StpMax) [inline docs/mindsdb-docs/docs/sql/create/databases.md:189]: Are these strings random? If not, then let put here something random

Comment (martyna-mindsdb) [reply docs/mindsdb-docs/docs/sql/create/databases.md:189]: I took examples primarily from the handler readme.md files, like this one: https://github.com/mindsdb/mindsdb/blob/staging/mindsdb/integrations/handlers/s3_handler/README.md

**Pull request merged**